contribs/gnodev/cmd/gnodev/main.go

```diff
@@ -23,7 +23,22 @@ func main() {
 			ShortHelp:  "Runs an in-memory node and gno.land web server for development purposes.",
 			LongHelp: `The gnodev command starts an in-memory node and a gno.land web interface, primarily for realm package development.
 
+Currently gnodev come with two mode <local> and <staging>, those command mostly differ by there default values, while gnodev local as default for working locally, satging default are oriented to be use on server.
```

Comment:
Run it through a grammar checker :)

```suggestion
Currently gnodev comes with two mode <local> and <staging>, those command mostly differ by there default values, while gnodev local as default for working locally, satging default are oriented to be use on server.
```